<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>538</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>9999</width>
    <height>9999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Player</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>645</width>
         <height>400</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>642</width>
          <height>464</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>640</width>
             <height>400</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QSlider" name="playback_slider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="prevmoment_button">
               <property name="font">
                <font>
                 <family>Sitka</family>
                 <pointsize>8</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Prev</string>
               </property>
               <property name="shortcut">
                <string>Left</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="slowdown_button">
               <property name="font">
                <font>
                 <family>Sitka Small</family>
                </font>
               </property>
               <property name="text">
                <string>&lt;&lt;</string>
               </property>
               <property name="shortcut">
                <string>Down</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="start_button">
               <property name="font">
                <font>
                 <family>Sitka Small</family>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>&gt;</string>
               </property>
               <property name="shortcut">
                <string>Space</string>
               </property>
               <property name="autoDefault">
                <bool>true</bool>
               </property>
               <property name="default">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="speedup_button">
               <property name="font">
                <font>
                 <family>Sitka Small</family>
                </font>
               </property>
               <property name="text">
                <string>&gt;&gt;</string>
               </property>
               <property name="shortcut">
                <string>Up</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="nextmoment_button">
               <property name="font">
                <font>
                 <family>Sitka</family>
                 <pointsize>8</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Next</string>
               </property>
               <property name="shortcut">
                <string>Right</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>154</width>
         <height>466</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="usesScrollButtons">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Alerts</string>
        </attribute>
        <widget class="QListWidget" name="listWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>181</width>
           <height>442</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Misc</string>
        </attribute>
        <widget class="QListWidget" name="listWidget2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>181</width>
           <height>442</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="changefile_action"/>
    <addaction name="change_videopath_action"/>
    <addaction name="separator"/>
    <addaction name="begin_analyze"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Изображение</string>
    </property>
    <addaction name="show_masks"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="show_masks">
   <property name="text">
    <string>Отображать маски</string>
   </property>
  </action>
  <action name="changefile_action">
   <property name="text">
    <string>Выбрать файл для анализа</string>
   </property>
  </action>
  <action name="change_videopath_action">
   <property name="text">
    <string>Указать путь к видеофайлам</string>
   </property>
  </action>
  <action name="begin_analyze">
   <property name="text">
    <string>Начать анализ</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
